src/Drupal/Translatable/Content/FieldCollectionTranslatable.php

```diff
@@ -51,8 +51,33 @@ public function getSourceLanguage() {
    */
   public function getTargetEntity($targetLanguage) {
     if (!isset($this->targetEntities[$targetLanguage]) || empty($this->targetEntities[$targetLanguage])) {
+      $target = $this->getRawEntity($this->entity);
```

Comment:
- [ ] Are these changes necessary, or can we avoid making changes here and punt until we/someone do #134?